nevergrad/benchmark/plotting.py

```diff
@@ -199,16 +199,19 @@ def create_plots(
     assert not any("Unnamed: " in x for x in df.columns), f"Remove the unnamed index column:  {df.columns}"
     assert "error " not in df.columns, f"Remove error rows before plotting"
     required = {"optimizer_name", "budget", "loss", "elapsed_time", "elapsed_budget"}
+    excluded = {x for x in set(df.columns) if x.startswith('{') and x.endswith('}')}  # ignore all descriptors starting enclosed in "{ }"
```

Comment:
instead of all these modifications, I would prefer you the columns directly from the dataframe as the first or second line of the function:
`df = df.loc[:, [x for x in df.columns if not x.startswith('{') and x.endswith('}')]`
This way the impact is really reduced, which is always better :D

Reply:
Done

Reply:
Done